README_submit.md

```diff
@@ -34,9 +34,10 @@ submission with flavors
 
 ##### with local slave
 		
-	spark-submit --class "mytest.spark.CloudantApp" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
-	spark-submit --class "mytest.spark.CloudantDF" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
-	spark-submit --class "mytest.spark.CloudantDFOption" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.10/spark_test_2.10-0.1-SNAPSHOT.jar
+	spark-submit --class "mytest.spark.CloudantApp" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantDF" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantDFOption" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
+	spark-submit --class "mytest.spark.CloudantStreaming" --master local[4] --jars cloudant-spark.jar examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar
```

Comment:
@mayya-sharipova I still don't get this spark_test_2.11-2.0.0.jar as expected

In the root folder I did:

`sbt clean package`

The output is: 
`[info] Packaging /Users/holger/dev/git/spark-cloudant/examples/scala/target/scala-2.11/spark_test_2.11-2.0.0.jar ...`

The target directory contains: 

`
23817 Sep 23 10:42 spark_test_2.11-2.0.0.jar
`